docs/geedocs/5.3.0/answer/3481558.html

```diff
@@ -330,7 +330,7 @@ <h3>Commands</h3>
 <p><i>Optional. </i>Specify the number of CPUs on this machine to use for processing. The default will be the maximum number of CPUs detected on the machine during installation. This command is available only in combination with --<code>assetroot</code>.</p>
 <h2>geselectpublishroot</h2>
 <pre>
-<code>geselectpublishroot <i>path</i></code> [--<strong>noprompt</strong>]</pre>
+<code>geselectpublishroot <i>path</i></code></pre>
```

Comment:
let's update copyright notice from `2018 Google, Open GEE Contributors` to `2018 Google, 2018-2019 Open GEE Contributors`